lib/still/compiler/incremental/node.ex

```diff
@@ -1,24 +1,25 @@
 defmodule Still.Compiler.Incremental.Node do
-  @doc """
-  A Incremental Node represents a file, or folder, that is processed
-  individually. Each file has a list of subscriptions and subcribers. The
-  subscriptions are the files included by the current file. The subscribers are
-  the files that the current file includes. When the current file changes, it
-  notifies the subscribers, and updates the subscriptions.
+  @moduledoc """
+  An incremental node represents a file, or folder, that is processed
+  individually.
+
+  Each file has a list of subscriptions and subcribers. The subscriptions are
+  the files included by the current file. The subscribers are the files that the
+  current file includes. When the current file changes, it notifies the
+  subscribers, and updates the subscriptions.
```

Comment:
It could be relevant to mention that a file may include another without subscribing to it. There's a boolean on the render function just to control that.

Reply:
It's documented in the `include` function